### DISCUSSION TITLE
Alexa via Haaska e AWS

### DISCUSSION THREAD

**Discussion by leonardus1973** [indomus/forum#1043] [HUB: Home Assistant]
buonasera,
mi sto cimentando a seguire la guida sul sito per collegare Alexa ad HA.
sono quasi convinto di aver fatto tutto correttamente ma al test di configurazione finale ricevo il seguente errore: "Task timed out after 3.00 seconds" - "Duration: 3003.92 ms	Billed Duration: 3000 ms	Memory Size: 128 MB	Max Memory Used: 22 MB". Sembra impieghi più di 3 secondi (forse quelli inclusi nel pacchetto free?).
riuscite a darmi dei suggerimenti sui passaggi da verificare?
grazie.

**Comment by snakuzzo** [comment]
hai IP pubblico ?
Hai configurato correttamente la URL nel [config.json](url) sulla lambda function su AWS ?

**Reply by leonardus1973** [reply]
ho un indirizzo su "https://myhome.duckdxxx.xxx:8123/api", ho inserito questo indirizzo nel file config.json.

**Reply by snakuzzo** [reply]
Se da internet accedi correttamente a quell'indirizzo, l'unico consiglio che possiamo darti è ricontrollare tutto minuziosamente dall'inizio.
Tutte le volte che l'integrazione non è andata è a causa di distrazioni e configurazioni errate

**Reply by leonardus1973** [reply]
A dire la verità da remoto accedo con: https://myhouse.duckxxx.org senza indicare la porta r ricevo un rrrore 404... Devo eliminarla anche dal config.json?

**Reply by snakuzzo** [reply]
E allora che c'entra la porta 8123 alla fine? :D
Nel config.json devi indicare `https://myhome.duckdns.org/api` ...
Devi usare la stessa url che usi per accedere da web (quindi da quello che dici senza porta) e con `/api` alla fine (è scritto chiaramemte nella guida come impostare). Riguardati tutta la guida perché la url va impostata anche in altri punti...non solo lì
